test/support/utils.h

```diff
@@ -123,6 +130,52 @@ is_equal_val(const T1& val1, const T2& val2)
     }
 }
 
+template <typename T, typename TOutputStream, typename = void>
+struct IsSupportStreamOutput : std::false_type
```

Comment:
nitpick: Maybe a name like `IsStreamInsertable` or `IsOutputStreamable` is a bit clearer.

Reply:
Fixed: renamed to `IsOutputStreamable`